src/ReinforcementLearningCore/src/core/stop_conditions.jl

```diff
@@ -23,8 +23,8 @@ struct ComposedStopCondition{S,T} <: AbstractStopCondition
     end
 end
 
-function (s::ComposedStopCondition)(args...)
-    s.reducer(sc(args...) for sc in s.stop_conditions)
+function stop(s::ComposedStopCondition, args...)
```

Comment:
Imho, the name `stop` does not convey what this function does. Can we perhaps rename this to `check_stop` or `check_stop_condition`?